style.css

```diff
@@ -1,14 +1,102 @@
-.title_3 {
-  color: rgb(0, 0, 0);
-  text-align: center;
+.page-portfolio {
+  background-image: url(images/bg-circle.png);
+  background-repeat: repeat;
+  background-attachment: fixed;
 }
 
-.section_info,
-.section_footer {
-  background: linear-gradient(90deg, #e7ecfb, #f3dbee);
+.page-portfolio-header,
+.page-portfolio-header-second {
+  font-family: monospace;
+  font-size: 22px;
+  color: #1e444f;
+  font-weight: 500;
+  letter-spacing: 2px;
+  text-align: justify;
+  margin: 10px;
+  padding: 10px;
+  line-height: 1.5;
+}
+
+.page-portfolio-header-name {
+  text-decoration: underline;
+  font-weight: bold;
+  text-align: justify;
+}
+
+.page-portfolio-header-image {
+  border-radius: 50%;
+  border: 3px solid #fff;
+  box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+  float: right;
+  margin: 0 50px 75px;
+  width: 210px;
+}
+
+.page-portfolio-header-info,
```

Comment:
if you have a case like this where you have a bunch of classes that all have the same css, that means that you dont need all of them. You can create one single class and use it instead of them.
When this grows and is 200+ lines of code, these extra class names will get confusing and increase the size of the css file without adding much value.
I would create a simple class like `header-text` and just use it instead of these

Reply:
It makes sense, fixed